gradle/dependencies.gradle

```diff
@@ -31,10 +31,12 @@ versions.jlayer = "1.0.1-gdx"
 versions.jorbis = "0.0.17"
 versions.junit = "4.11"
 versions.appleExtension = "1.4"
-versions.androidPlugin = "3.1.0"
+versions.androidPlugin = "3.5.4"
 versions.multiDex = "2.0.1"
-versions.androidSdk = 29
-versions.androidBuildTools = "28.0.3"
+versions.androidCompileSdk = 30
+versions.androidTargetSdk = 29
+versions.androidMinSdk = 14
+versions.androidBuildTools = "29.0.3"
```

Comment:
I think it would be better to have build tools and compile sdk on the same version, but it works this way without warnings.

Reply:
Interestingly, it hasn't been required to specify a build tools version in quite some time. The tools will automatically choose the minimum required for the plugin. One less thing to have to keep track of.

Reply:
As @MobiDevelop says it's not required anymore to explicitly set it because it automatically picks the minimum supported required by the Android gradle plugin (in the case of `3.5.3` the minimum build tools is `28.0.3` (https://developer.android.com/studio/releases/gradle-plugin)). My understanding is that if we want to use a different one like in this case, as long as we use a version higher than the minimum , it should be ok independently of the `compileSDK`. I'm ok removing it alltogether or updating it to a different version, I see the advantage of not having to deal with another version (and let it uise 28.0.3) but there may be a better argument for using the latest.